roles/ara_api/tasks/wsgi_server/gunicorn.yaml

```diff
@@ -40,6 +40,7 @@
       copy:
         src: ara-gunicorn.te
         dest: "{{ ara_api_root_dir }}/ara-gunicorn.te"
+        mode: 0444
```

Comment:
Out of curiosity: any particular reason for 0444 ? The default umask tends to set regular files to 644 which would be fine here.

Reply:
No, just another gentle reminder that the file is supposed to be readonly ;-)